benchmarks/add.py

```diff
@@ -15,12 +12,12 @@ class Add(BaseBench):
 
     def setup(self, link_type):
         super().setup()
+
         self.init_git()
         self.init_dvc()
-        dataset_path = os.path.join(
-            os.environ["ASV_CONF_DIR"], "data", "cats_dogs"
-        )
-        shutil.copytree(dataset_path, "data")
+
+        shutil.copytree(CATS_DOGS_DIR, "data")
```

Comment:
so maybe just
```
self.gen("data", template="cats_and_dogs")
```
?

Feels weird that you have to import two things for writing this single line.